src/main/webapp/resources/js/components/sequence-files/SingleEndFileRenderer.jsx

```diff
@@ -16,6 +16,7 @@ import { useSelector } from "react-redux";
  * @function get file processing state function
  * @param qcEntryTranslationKeys
  * @param displayConcatenationCheckbox Whether to display checkbox or not
+ * @function set default sequencing object for sample
```

Comment:
Again!  Stop using this :)

Reply:
NEVER! Updated in a23c555bf4b038f2ee0e43742e27ff837b87eb78